src/components/settings/SettingsLMSTab/LMSConfigs/BlackboardConfig.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Form, useToggle } from '@edx/paragon';
+import { buttonText, handleErrors } from '../LMSConfigPage';
+import LmsApiService from '../../../../data/services/LmsApiService';
+import { snakeCaseDict } from '../../../../utils';
+import ConfigError from '../ConfigError';
+
+const BlackboardConfig = ({ id, onClick }) => {
+  const [clientId, setClientId] = React.useState('');
+  const [clientSecret, setClientSecret] = React.useState('');
+  const [blackboardBaseUrl, setBlackboardBaseUrl] = React.useState('');
+  const [isOpen, open, close] = useToggle(false);
+
+  const config = {
+    clientId,
+    clientSecret,
+    blackboardBaseUrl,
+  };
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    const transformedConfig = snakeCaseDict(config);
+    // this will need to change based on save draft/submit
+    transformedConfig.active = false;
+    transformedConfig.enterprise_customer = id;
+    let err;
+    try {
+      await LmsApiService.postNewBlackboardConfig(transformedConfig);
+      err = undefined;
+    } catch (error) {
+      err = handleErrors(error);
+    } if (err) {
+      open();
+    } else {
+      onClick();
+    }
+  };
+
+  return (
+    <span data-test="BlackboardConfig">
+      <ConfigError isOpen={isOpen} close={close} />
+      <Form data-test="form">
+        <Form.Group>
```

Comment:
This applies to all the config forms, but in testing locally I noticed each field extends the across the whole window. As discussed offline, could we add a bumper so it matches the figma mock?